src/view/scripts.inc

```diff
@@ -0,0 +1,48 @@
+<!-- Namespace for Skosmos variables -->
```

Comment:
I would make this a little bit more descriptive, e.g. 

    <!-- Namespace for Skosmos variables passed from the backend to the frontend code -->